zebra-rpc/src/methods/types/get_block_template.rs

```diff
@@ -918,46 +897,37 @@ pub fn standard_coinbase_outputs(
     let miner_reward =
         miner_reward.expect("reward calculations are valid for reasonable chain heights");
 
-    combine_coinbase_outputs(funding_streams, miner_address, miner_reward, like_zcashd)
-}
-
-/// Combine the miner reward and funding streams into a list of coinbase amounts and addresses.
-///
-/// If `like_zcashd` is true, try to match the coinbase transactions generated by `zcashd`
-/// in the `getblocktemplate` RPC.
-fn combine_coinbase_outputs(
-    funding_streams: HashMap<FundingStreamReceiver, (Amount<NonNegative>, &transparent::Address)>,
-    miner_address: &transparent::Address,
-    miner_reward: Amount<NonNegative>,
-    like_zcashd: bool,
-) -> Vec<(Amount<NonNegative>, transparent::Script)> {
     // Collect all the funding streams and convert them to outputs.
     let funding_streams_outputs: Vec<(Amount<NonNegative>, &transparent::Address)> =
         funding_streams
             .into_iter()
             .map(|(_receiver, (amount, address))| (amount, address))
             .collect();
 
+    // Combine the miner reward and funding streams into a list of coinbase amounts and addresses.
     let mut coinbase_outputs: Vec<(Amount<NonNegative>, transparent::Script)> =
         funding_streams_outputs
             .iter()
-            .map(|(amount, address)| (*amount, address.create_script_from_address()))
+            .map(|(amount, address)| (*amount, address.script()))
             .collect();
 
+    let script = miner_address
+        .to_transparent_address()
+        .expect("address must have a transparent component")
```

Comment:
It _may_ be useful to leave a TODO here if you want.

Reply:
This will certainly go away soon.